kratos/solving_strategies/schemes/residual_based_bossak_displacement_scheme.hpp

```diff
@@ -160,84 +160,68 @@ class ResidualBasedBossakDisplacementScheme: public Scheme<TSparseSpace,TDenseSp
     }
 
     /**
-     * Performing the update of the solution
-     * Incremental update within newton iteration. It updates the state variables at the end of the time step: u_{n+1}^{k+1}= u_{n+1}^{k}+ \Delta u
-     * @param rModelPart: The model of the problem to solve
-     * @param rDofSet: Set of all primary variables
-     * @param A: LHS matrix
-     * @param Dx: incremental update of primary variables
-     * @param b: RHS Vector
+     * @brief Performing the update of the solution
+     * @details Incremental update within newton iteration. It updates the state variables at the end of the time step u_{n+1}^{k+1}= u_{n+1}^{k}+ \Delta u
+     * @param rModelPart The model of the problem to solve
+     * @param rDofSet Set of all primary variables
+     * @param A LHS matrix
+     * @param Dx incremental update of primary variables
+     * @param b RHS Vector
      */
 
     void Update(
         ModelPart& rModelPart,
         DofsArrayType& rDofSet,
         TSystemMatrixType& A,
         TSystemVectorType& Dx,
-        TSystemVectorType& b ) override
+        TSystemVectorType& b 
+        ) override
     {
         KRATOS_TRY;
 
-        // std::cout << " Update " << std::endl;
-
-        const unsigned int NumThreads = OpenMPUtils::GetNumThreads();
-
         // Update of displacement (by DOF)
-        OpenMPUtils::PartitionVector DofPartition;
-        OpenMPUtils::DivideInPartitions(rDofSet.size(), NumThreads, DofPartition);
-
-        const int ndof = static_cast<int>(rDofSet.size());
-        typename DofsArrayType::iterator DofBegin = rDofSet.begin();
+        const int num_dof = static_cast<int>(rDofSet.size());
```

Comment:
I think the static_cast is unnecessary here.

Reply:
It is, I checked, is is std::size_t type

Reply:
ok